app/controllers/cards_controller.rb

```diff
@@ -16,6 +16,7 @@ def edit; end
   def create
     @card = Card.new(card_params)
     @card.user_id = current_user.id
+    @card.pack_id = current_user.current_pack_id if current_user.current_pack_id.present?
```

Comment:
it is not possible to create card without pack. if pack does not exist, you should create it.